src/pages/mission/quiz/components/QuizCard.tsx

```
@@ -0,0 +1,68 @@
+    <div className="flex w-full flex-col rounded-2xl bg-white px-18 pt-30 pb-32">
+      <div className="flex flex-col items-center justify-center px-10">
+        <p className="body-2 whitespace-pre-line text-center text-black">
+          {question.questionText}
+        </p>
+      </div>
+
+      <div className="relative flex w-full items-center justify-center pt-49">
+        <div className="inline-flex w-full gap-4 px-12">
+          {Array.from({ length: 10 }).map((_, index) => (
+            <div
+              key={index}
```

Biome (2.3.13)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)